client/tx/search.go

```diff
@@ -62,9 +64,8 @@ $ gaiacli tendermint txs --tag test1,test2 --any
 	}
 
 	cmd.Flags().StringP(client.FlagNode, "n", "tcp://localhost:26657", "Node to connect to")
-
-	// TODO: change this to false once proofs built in
-	cmd.Flags().Bool(client.FlagTrustNode, true, "Don't verify proofs for responses")
+	cmd.Flags().Bool(client.FlagTrustNode, false, "Don't verify proofs for query responses")
```

Comment:
Seems we're inconsistent with this flag's description. Can we update all occurrences to something like: `Trust connected full node (don't verify proofs for responses)`?